protocols/grpc/agent.proto

```diff
@@ -426,3 +427,10 @@ message StringUser {
 	string gid = 2;
 	repeated string additionalGids = 3;
 }
+
+message UploadFileRequest {
+	string path = 1;
+	string content = 2;
```

Comment:
I think it'd make more sense to have this as an array of bytes instead of a string since it represents the content of the file.
And I'd use an extra field holding the size of this byte array.